src/pm-rpc/rpc.js

```diff
@@ -7,6 +7,9 @@ import {send, sendResponse} from './privates/messageManager'
 import * as messageHandler from './privates/messageHandler'
 import {serialize as serializeError} from './privates/errorSerializer'
 
+// eslint-disable-next-line no-undef, lodash/prefer-lodash-typecheck, no-use-before-define
```

Comment:
why we need all the ignores ?

Reply:
* `lodash/prefer-lodash-typecheck` is needed b/c we can't access the variable when it does not declared
* `eslint-disable-next-line no-undef` is needed b/c I prefer not to set the target environment to node, as the entire codebase is in a web env. it's just for the `require` access